section1/reflection.md

```diff
@@ -1,19 +1,39 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+I tend to wait too long. Asking questions causes me a *lot* of anxiety, for all the reasons listed in the articles. These are some of my personal goals for this program: get more comfortable not knowing things, not being "the best", and asking questions that might reveal all of the above.
```

Comment:
Fantastic habit to start building at Turing because it's an important skill in this industry. I'm still battling the anxiety of asking questions myself ~1 year out of the program. I like your realistic goal to get more comfortable asking questions 🙂